src/Chirp.DB/DBFacade.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Diagnostics;
+using Microsoft.Data.Sqlite;
+
+namespace Chirp.DB;
+
+public class DBFacade
+{
+    SqliteConnection connection;
+    
+    // Establish connection
+    public DBFacade() {
+        string dbPath = Path.Combine(Path.GetTempPath(), "chirp.db");
+        string? dbPathEnvVar = Environment.GetEnvironmentVariable("CHIRPDBPATH");
+
+        if(!String.IsNullOrEmpty(dbPathEnvVar))
+        {
+            dbPath = dbPathEnvVar;
+        }
+
+        // Execute init.sh in /scripts **TEMPORARY**
+        var initScriptPath = Path.Combine(Directory.GetCurrentDirectory(), "scripts", "init.sh");
+        var process = new Process
+        {
+            StartInfo = new ProcessStartInfo
+            {
+                FileName = "/bin/bash",
+                Arguments = initScriptPath,
+                RedirectStandardOutput = true,
+                UseShellExecute = false,
+                CreateNoWindow = true
+            }
```

Comment:
Yea this is just a temporary solution until we deploy it to azure

Reply:
Should be fixed now. Use the environment variable `CHIRPDBPATH` when deploying to azure to define the path for the db. If no db exists at `CHIRPDBPATH` already, it will create one at the specified location and fill it with some dummy data.